documentation/overview.md

```diff
@@ -69,33 +70,35 @@ Next to get everything set up all node module dependencies must be installed.
 To do this run
 
 ```bash
-npm install && cd client && npm i && cd services/server-service && npm i
+npm install && cd client && npm i && cd services/server && npm i
 ```
 
-in the `/services/server-service` directory create a `.env` file
+in the `/services/server` directory create a `.env` file
 
 and paste
 
 ```ts
-MONGO_USERNAME = '';
-MONGO_PASSWORD = '';
+MONGO_USERNAME =
+MONGO_PASSWORD =
+MONGO_DB_STRING =
 
-SERVER_PORT = '';
+SERVER_PORT =
 ```
```

Comment:
Was this what you meant to put here? I think this is even less valid syntax than before?

Maybe something like this:
```ts
MONGO_USERNAME = '<mongo username>';
MONGO_PASSWORD = '<password>';
MONGO_DB_STRING = '<db name>';
SERVER_PORT = '<server port>';
```

idk

Reply:
if you copy paste the `;` is won't work so I was trying to dummy proof it